tests/test_all.py

```diff
@@ -73,14 +88,32 @@ def test_product(auto_container):
     :file:`/etc/products.d/baseproduct` is a link to it
     """
     assert auto_container.connection.file("/etc/products.d").is_directory
-    assert auto_container.connection.file("/etc/products.d/SLES.prod").is_file
-    assert auto_container.connection.file(
-        "/etc/products.d/baseproduct"
-    ).is_symlink
-    assert (
-        auto_container.connection.file("/etc/products.d/baseproduct").linked_to
-        == "/etc/products.d/SLES.prod"
-    )
+    if OS_VERSION == "latest":
+        assert auto_container.connection.file(
+            "/etc/products.d/openSUSE.prod"
+        ).is_file
+        assert auto_container.connection.file(
+            "/etc/products.d/baseproduct"
+        ).is_symlink
+        assert (
+            auto_container.connection.file(
+                "/etc/products.d/baseproduct"
+            ).linked_to
+            == "/etc/products.d/openSUSE.prod"
+        )
+    else:
+        assert auto_container.connection.file(
+            "/etc/products.d/SLES.prod"
+        ).is_file
+        assert auto_container.connection.file(
+            "/etc/products.d/baseproduct"
+        ).is_symlink
+        assert (
+            auto_container.connection.file(
+                "/etc/products.d/baseproduct"
+            ).linked_to
+            == "/etc/products.d/SLES.prod"
+        )
```

Comment:
Wouldn't this be a bit simpler?

```suggestion
    product_file = f"/etc/products.d/{'openSUSE.prod' if OS_VERSION == 'latest' else 'SLES.prod'}"

    assert auto_container.connection.file(product_file).is_file
    assert auto_container.connection.file(
            "/etc/products.d/baseproduct"
    ).is_symlink
    assert (
            auto_container.connection.file(
                "/etc/products.d/baseproduct"
            ).linked_to
            == product_file
    )
```